src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemControl.xaml

```diff
@@ -87,8 +87,11 @@
                     Margin="4,12,4,0"
                     VerticalAlignment="top"
                     Visibility="{Binding CheckBoxesEnabled, RelativeSource={RelativeSource AncestorType={x:Type nuget:InfiniteScrollList}}, Converter={StaticResource NonVisualStateHiddenBooleanToVisibilityConverter}}"
+                    Focusable="False"
                     AutomationProperties.Name="{x:Static nuget:Resources.CheckBox_Selected}"
-                    IsChecked="{Binding Selected}" />
+                    IsChecked="{Binding Selected}"
+                    Checked="CheckBox_Checked"
```

Comment:
Since this is used to handle both Checked and Unchecked, the name or suffix could be non-specific to the check/uncheck action and be something like ToggleChecked or OnToggleChecked

Reply:
Done. Went with `CheckBox_Toggled`.